integrated_channels/moodle/client.py

```diff
@@ -150,6 +151,100 @@ def _get_access_token(self):
                 response.status_code
             )
 
+    @moodle_request_wrapper
+    def _get_enrolled_users(self, course_id):
+        """
+        placeholder
```

Comment:
is this text intentional or actually means the field is a placeholder?

Reply:
it means I was rushing to get the PR up and said to myself "oh I'll just add the doc strings soon" 😬

Reply:
annnnd replaced!